src/Nelmio/Alice/Instances/Instantiator/Methods/ReflectionWithoutConstructor.php

```diff
@@ -22,6 +22,10 @@ public function canInstantiate(Fixture $fixture)
     {
         $reflConstruct = new \ReflectionMethod($fixture->getClass(), '__construct');
 
+        if (!$reflConstruct->isPublic() && '__construct' !== $fixture->getConstructorMethod()) {
```

Comment:
@gquemener Let's just add this conditional bit to the line below - it's effectively the same thing, but this makes it look like there's much more change happening than there actually is. Make sense?